kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/process/Rules.java

```diff
@@ -0,0 +1,5 @@
+package com.kodilla.good.patterns.food2door.process;
+
+public class Rules {
```

Comment:
You should focus on main purpose of this task: it was about enabling adding new Companies to the Food2Door offer - there is too much additional code (like Rules here). Keep it simple :)